ctpolicy/ctpolicy.go

```diff
@@ -56,15 +56,32 @@ func (group *LogGroupInfo) populate(ll *loglist.LogList, included func(log *logl
 	}
 }
 
+// LogPolicyData contains info on log-partition and submission requirements for a single cert. Key always matches value Name field.
+type LogPolicyData map[string]*LogGroupInfo
```

Comment:
One step to shorten this PR would be to factor out introducing this type into a separate PR. That would exclude policy files like `applepolicy.go` almost entirely.

Reply:
Extracted to PR-426.

Reply:
Btw, you can use #426 syntax, it automatically creates a link, and shows up in both PR's histories as a relation.